src/modules/search.js

```diff
@@ -142,10 +158,12 @@ class Search {
    * @param {string} [parameters.sortBy='relevance'] - The sort method for results
```

Comment:
Can we add `offset` here? (not here, after page but GH doesn't let me add a comment there)